### PR TITLE
Add remove edit feature

### PR DESCRIPTION
**Pull request by trace-kadenyi**
Update the To-Do-List to add the following functionalities using JavaScript:
       - Add tasks to the list
       - Remove tasks from the list
       - Edit tasks in the list
       - Save the list and changes in local storage.

### CODE REVIEW

File: src/modules/todoObjectClass.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      completed: this.completed,
+    });
+    // save to local storage
+    localStorage.setItem('todotasks', JSON.stringify(mainArr));
+    // update the length of the container
+    createTodoListComponents(this, mainArr.length + 1);
+  }
+
+  // remove
+  removeFromToDo(index) {
```
Comment (omarsalem7) [inline src/modules/todoObjectClass.js:32]: - [ ] I explained in [this video](https://www.loom.com/share/cf74eb7f322f49f0bb52321a46aae0f4), what is the wrong behavior happened when I try to remove todo, kindly fix these bugs.

File: src/modules/todoObjectClass.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        editItem = (id, inputValue) => {
+          // loop through array of objects and adjust task description
+          for (let i = 0; i < MyToDo.tasks.length; i += 1) {
+            if (MyToDo.tasks[i] === id) {
+              MyToDo.tasks[i].description = inputValue;
+            }
+          }
+          // save changes to local storage
+          localStorage.setItem('todotasks', JSON.stringify(MyToDo.tasks));
+        }
```
Comment (omarsalem7) [inline src/modules/todoObjectClass.js]: - [ ] Also consider after editing todo and refreshing the page the updated todo should be saved and displayed not vanished.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @TracK92 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/todoObjectClass.js (hunk 2 above)
Comment (trace-kadenyi) [reply src/modules/todoObjectClass.js]: [omarsalem7](https://github.com/omarsalem7) thank you for your review and especially for the video. I will work on making the changes.

**Comment by trace-kadenyi** [comment]
Hi, 

I was finally able to fix the bugs in my code as highlighted by the previous reviewer. Kindly re-review my code. Thank you.

**Review by mwanawabangona — APPROVED** [review]
Hi @TracK92 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Solid pull request scroll
* ✅  Good quality commits
* ✅  Implemented Changes as requested 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @mwanawabangona in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**